Core/GDCore/Extensions/Builtin/CameraExtension.cpp

```diff
@@ -31,7 +33,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsCameraExtension(
           _("Camera center X position"),
           _("the X position of the center of a camera"),
           _("the X position of camera _PARAM4_ (layer: _PARAM3_)"),
-          _("Layers and cameras"),
+          "",
           "res/conditions/camera24.png")
```

Comment:
if the group for the extension already has this icon defined, is it needed to define it per instruction?

Reply:
Ideally no, we would define only the group icon and the instruction would inherit from it. But this is not how it works today. 
We could at least rework this in the future so that the default icon for all instructions/expressions are the extension icon but I've not done it because it's a lot of work.
But that would be neat, we would remove 2 lines for each instruction, and could still keep specific icons for some instructions.